### PR TITLE
Add `loss-agg-mode` for policy loss

### PR DESCRIPTION
**Pull request by hiyuchang**
## Description

As the title say.s

## Checklist

Please check the following items before code is ready to be reviewed.

- [x]  Code has passed all tests
- [x]  Docstrings have been added/updated in Google Style
- [x]  Documentation has been updated
- [x]  Code is ready for review

### CODE REVIEW

File: trinity/algorithm/policy_loss_fn/chord_policy_loss.py [hunk 1]
```diff
@@ -49,9 +49,14 @@ def __call__(  # type: ignore
     ) -> Tuple[torch.Tensor, Dict]:
         token_prob = torch.exp(logprob)
         if self.use_token_level_loss:
-            sft_loss = masked_mean(-logprob * token_prob.detach(), action_mask)
+            sft_loss = masked_loss(
+                -logprob * token_prob.detach(), action_mask, loss_agg_mode="token-mean"
+            )
         else:
-            sft_loss = masked_mean(-logprob * token_prob.detach(), action_mask, axis=1).mean()
+            sft_loss = masked_loss(
```
Comment (pan-x-c) [inline trinity/algorithm/policy_loss_fn/chord_policy_loss.py]: Add `loss_mode` to the init function and deprecate `use_token_level_loss`

File: trinity/algorithm/utils.py [hunk 2]
```diff
@@ -6,6 +6,48 @@
 import torch
 
 
+def masked_loss(values, mask, loss_agg_mode="token-mean", normalizer=None):
+    """
+    Compute loss from values and mask with various aggregation modes.
```
Comment (yanxi-chen) [inline trinity/algorithm/utils.py:11]: add reference to verl
